doc/classes/Callable.xml

```diff
@@ -69,24 +69,28 @@
 			<return type="int">
 			</return>
 			<description>
+				Returns the hash value of this [Callable]'s object.
 			</description>
 		</method>
 		<method name="is_custom">
 			<return type="bool">
 			</return>
 			<description>
+				Returns true if [Callable] is a custom callable thats behaviour is different based on implementation details.
```

Comment:
```suggestion
				Returns true if [Callable] is a custom callable whose behaviour differs based on implementation details.
```

Also, I don't find that description isn't very clear. Can it be made more detailed?

Reply:
I know, but there is literally no information about CustomCallables (none evident from the source). My guess is that you use it when you don't have an object pointer or object id and/or need some sort of custom behaviour. Also, get_custom is not exposed to script, so I guess the method is almost useless in a script level. Any ideas?

Reply:
I think the whole purpose of the method (without the ability to get the CustomCallable) is to state whether or not `get_method` can be used.